src/server/family_utils.cc

```diff
@@ -0,0 +1,45 @@
+#include "server/family_utils.h"
+
+#include "base/logging.h"
+
+namespace dfly {
+
+sds WrapSds(std::string_view s) {
+  static __thread sds tmp_sds = nullptr;
+  if (tmp_sds == nullptr)
+    tmp_sds = sdsempty();
```

Comment:
```suggestion
  static __thread sds tmp_sds = sdsempry();
```

Reply:
@dranikpg I did not approve since Roman did. If you apply this change just let me know and will accept quickly